packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js

```diff
@@ -25,11 +25,19 @@ class SettingsContainer extends React.PureComponent {
     this.props.updateUI({ updateToggled: !this.props.ui.updateToggled })
   }
 
+  validateCurrentPassword = (value) =>
+    value === this.props.currentWalletPassword ? undefined : 'Incorrect password'
+
+  validatePasswordConfirmation = (value, allValues) =>
+    (value === allValues.newPassword) ? undefined : 'Passwords do not match'
+
```

Comment:
It seems we're keeping many of our validators in `services/FormHelper`, could you move those there for consistency and translations?

Reply:
Sure